src/configuration/app/variables.md

```diff
@@ -0,0 +1,115 @@
+# Variables
+
+Platform.sh provides a number of ways to set [environment variables](/development/variables.md), either globally or specific to a single environment.  For values that should be consistent between different environments (because they're configuring the application or runtime itself, generally) the easiest way to control them is to set them in the `.platform.app.yaml` file.
```

Comment:
I've deliberately kept those two terms slightly separate based on my conversations with Fabien.  Calling values that only end up in $PLATFORM_VARIABLES "environment variables" is subtly incorrect.  But there's also Unix environment variables vs. variables defined per-environment, which are also "environment variables" in English but a different meaning of "environment".

Or are you referring to just the first line in the link text?